spynnaker8/setup-pynn.py

```diff
@@ -1,9 +1,9 @@
 import pyNN
 import os
 
-if not pyNN.__version__.startswith("0.8"):
+if not pyNN.__version__.startswith("0.9"):
     raise Exception(
```

Comment:
The rest of the code appears to support both 0.8 and 0.9…

Reply:
While it is true that the code does support pyNN 0.8 for any users running setup we should enforce pyNN 0.9.  as pyNN 0.8 is depricated by the pyNN developers.